content/docs/5.1/deploy/operate/upgrade.md

```diff
@@ -38,18 +37,18 @@ Before upgrading your TiKV cluster, you first need to upgrade TiUP.
 Skip this step if one of the following situations applies:
 
 + You have not modified the configuration parameters of the original cluster. Or you have modified the configuration parameters using `tiup cluster` but no more modification is needed.
-+ After the upgrade, you want to use v5.0's default parameter values for the unmodified configuration items.
++ After the upgrade, you want to use default parameter values of v5.0 for the unmodified configuration items.
 {{< /info >}}
 
-1. Edit the topology file:
+1. Enter the `vi` editing mode to edit the topology file:
```

Comment:
I am not sure whether the following command opens the vi editor. The vi editor is usually opened by `vi` or `vim` command.

Reply:
Your point makes sense. But I think the command provided here is to edit the file, not to enter the vi editor.

Reply:
Maybe we can change the sentence to "Edit the topology file in 'vi' editing mode"?